src/Avalonia.Base/Platform/Storage/IStorageBookmarkItem.cs

```diff
@@ -1,8 +1,14 @@
-﻿using System.Threading.Tasks;
+﻿using System.IO;
+using System.Threading.Tasks;
 using Avalonia.Metadata;
 
 namespace Avalonia.Platform.Storage;
 
+internal interface IStorageItemWithFileSystemInfo : IStorageItem
+{
+    FileSystemInfo FileSystemInfo { get; }
```

Comment:
Is this _guaranteed_ to be always available? Otherwise mark as `?`

Reply:
Ah, nvm, it's internal

Reply:
On platforms where this interface is implement - yes, it's guaranteed.
It's used as a shortcut when resolving TryGetLocalPath, to avoid `string -> Uri -> string` conversion in providers that use FileSystemInfo.